tests/tas-elasticsearch/src/test/resources/test-suites/elasticsearch-parallel-suite.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="Parallel REST API tests for Elasticsearch" verbose="1" parallel="true" thread-count="100">
```

Comment:
Why thread count is set to 100, it's an upper limit?

Reply:
Oh - this was an accident. I meant to leave it at the default (5), but I was playing around with it locally.